src/tasks.js

```diff
@@ -52,10 +58,18 @@ export const getTasksArray = () => {
     tasksArray = [taskOne, taskTwo, taskThree];
   } else {
     tasksArray = JSON.parse(localStorage.getItem('tasksArray'));
+    tasksArray = parseTasks(tasksArray);
   }
   return tasksArray;
 };
 
 export const setTasksArray = (newTasksArray) => {
   localStorage.setItem('tasksArray', JSON.stringify(newTasksArray));
 };
```

Comment:
- [ ] Every time you add a new element to the tasks array, you are supposed to provide a new index number for it. Please make sure to have a counter array for this. Also, make sure to update the counter array every time the length of the tasks array changes. That is: every time you add an element, the counter array should increase, and every time you delete an element, the counter array should decrease. This is done so that the next element you add to the array has a correct index value.

Reply:
A new method has been created to satisfy this requirement, thanks 👍